elastic/kibana
# [Ingest Manager] add _meta field to index templates

https://github.com/elastic/kibana/issues/70235

Adds the `_meta` field to index templates created through the Ingest Manager.

> We have two use cases that would benefit from adding metadata to our ES assets like index templates. The first is to identify assets we create to ES UI so we can provide users a visual indicator that these assets are managed by Ingest Manager. We can also provide a warning to users who attempt to change those assets that things could change when an integration package is updated. #69573. The second use case is for telemetry on which integrations are most popular in terms of data generated. We are aiming to get this in 7.9 since some of the telemetry work depends on it.

eg:

```
_meta: {
      package: {
         name: 'system',
       },
      managed_by: 'ingest-manager',
    }
```

# Test

- Install a package
- Go to dev tools and run `GET /_index_template/`
- find index templates associated with the package you installed and check the field exists
